doc/source/report/classification.rst

```diff
@@ -13,18 +13,19 @@ mouse data, since experimenting directly on human is difficult.
 The important concern is whether we can classify different strains of
 mice based on mouse behaviors through machine learning algorithms.
 Important features in classification models will be extracted for future
-study. Another question of interest is on whether genetic affects
-(different strain effects) psychological disorders such as depression,
-or overeating. The fundamental hypothesis is that different mouse
+study. The fundamental hypothesis is that different mouse
 strains have different behaviors, and we are able to use these behaviors
-as features to classify mice successfully.
+as features to classify mice successfully. On the other hand, we are also 
+trying to use unsupervised clustering algorithms to cluster mice, and 
+ideally different strains should have different clustering distributions.
 
-Statement of statistical problems
+
+Statement of Statistical Problems
 ---------------------------------
 
 The researchers design the experiment as follow: they have 16 different
```

Comment:
Hey guys! Jarrod has asked me to help review the wording of some of the reports, so I'll make some comments on things I see below. Feel free to take them or leave them, they're just suggestions. Hopefully you guys don't see this as imposing!
-Alanna

"follow" should be "follows" in this line